packages/legacy/core/App/navigators/OnboardingScreens.ts

```diff
@@ -0,0 +1,140 @@
+import { TFunction } from 'i18next'
+import { TransitionPresets, StackNavigationOptions, StackScreenProps } from '@react-navigation/stack'
+import { ParamListBase, RouteConfig, StackNavigationState } from '@react-navigation/native'
+import { Screens, ScreenOptionsType } from '../types/navigators'
+import type { StackNavigationEventMap } from '@react-navigation/stack/lib/typescript/src/types'
+
+type ScreenOptions = RouteConfig<
+  ParamListBase,
+  Screens,
+  StackNavigationState<ParamListBase>,
+  StackNavigationOptions,
+  StackNavigationEventMap
+>
+
+interface ScreenComponents {
+  Splash: React.FC<StackScreenProps<ParamListBase>>
+  Preface: React.FC<StackScreenProps<ParamListBase>>
+  Terms: React.FC<StackScreenProps<ParamListBase>>
+  NameWallet: React.FC<StackScreenProps<ParamListBase>>
+  useBiometry: React.FC<StackScreenProps<ParamListBase>>
+  PushNotification: React.FC<StackScreenProps<ParamListBase, Screens.UsePushNotifications>>
+  Developer: React.FC<StackScreenProps<ParamListBase>>
+  AttemptLockout: React.FC<StackScreenProps<ParamListBase>>
+  OnBoardingScreen: React.FC
```

Comment:
Should we take the opportunity to change this while we're touching this code, so it's the same casing as it is everywhere else?
```suggestion
  OnboardingScreen: React.FC
```

Reply:
Fixed.